docs/source/cli_help/anno.rst

```diff
@@ -26,6 +26,7 @@ To allow GitHub API access for private repositories (and also to perform side-ef
 a GitHub API token with ``repo`` scope is required, see https://github.com/settings/tokens. This will be resolved from the first of:
```

Comment:
Actually, while you're here: there's an `include` feature in .rst (see e.g. cli.rst)... maybe worth using for this repetetive section? already repeated in 3 places in .rst